src/entities/Updates/routes.ts

```diff
@@ -66,13 +67,14 @@ async function getProposalUpdates(req: Request<{ proposal: string }>) {
 }
 
 async function createProposalUpdate(req: WithAuth<Request<{ proposal: string }>>) {
-  const { health, introduction, highlights, blockers, next_steps, additional_notes } = req.body
+  const { author, health, introduction, highlights, blockers, next_steps, additional_notes } = req.body
 
   const user = req.auth!
   const proposalId = req.params.proposal
   const proposal = await ProposalModel.findOne<ProposalAttributes>({ id: proposalId })
+  const isAuthorOrCoauthor = user && (proposal?.user === user || isCoauthor(proposalId, user)) && author === user
```

Comment:
I think you have to await `isCoauthor` here 🤔

Reply:
Yes thanks, my bad!